docs/community/topics/i18n.rst

```diff
@@ -1,120 +1,112 @@
+positions (line numbers) of the translatable strings.
+Updating positions is optional - the line numbers are to inform the human translator, not to perform the translation.
 
-If you *only* change non-translatable text (like HTML markup), the `extract` and `update` commands will only update the positions (line numbers) of the translatable strings. Updating positions is optional - the line numbers are to inform the human translator, not to perform the translation.
+If you change translatable strings, the ``extract`` command will extract the new or updated strings to the ``POT`` file,
+and the ``update`` command will try to perform a fuzzy match between the new or updated strings and existing translations in the ``PO`` files.
+If there is a fuzzy match, a comment like ``#, fuzzy`` is added before the matched entry,
+this means that the translation needs to be manually reviewed and possibly updated.
+If after reviewing the translation you decide to keep the existing translation, you can remove the ``#, fuzzy`` comment from the entry.
+Otherwise, it will add a new translation entry.
 
-If you change translatable strings, the `extract` command will extract the new or updated strings to the POT file, and the `update` command will try to fuzzy match the new or updated strings with existing translations in the PO files.
-If there is a fuzzy match, a comment like `#, fuzzy` is added before the matched entry.
-Otherwise, a new entry is added and needs to be translated.
+.. _adding-new-language:
 
+Adding a new language
+----------------------
 
-.. _translating-the-theme:
+The list of currently supported languages is available in the ``src/pydata_sphinx_theme/locale`` directory.
 
-Add translations to translateable text
---------------------------------------
+To add a new language, follow these steps:
 
-Once text has been marked as translateable, and ``PO`` files have been generated for it, we may add translations for new languages for the phrase.
-This section covers how to do so.
+#. Identify the `ISO 639-1 code <https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes>`__ for the new language.
 
-.. note::
-
-   These steps use the Spanish language as an example.
-   To translate the theme to another language, replace ``es`` with the language's two-letter lowercase `ISO 639-1 code <https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes>`__.
-
-#. If the language's code matches no sub-directory of the `pydata_sphinx_theme/locale <https://github.com/pydata/pydata-sphinx-theme/tree/main/pydata_sphinx_theme/locale>`__ directory, initialize the language's message catalog (PO file) with `PyBabel init <https://babel.pocoo.org/en/latest/cmdline.html#init>`__:
+#. Make a copy of the ``POT`` file for this new language:
 
    .. code-block:: bash
+      
+      # for example, to add Quechua (ISO 639-1 code: qu)
+      tox -e i18n-new-locale -- qu
```

Comment:
```suggestion
      tox -e i18n-new-locale -- qu

#. Start translating the theme's text into the newly added language (see :ref:`translating-the-theme`).
```

I know it's the immediate next section, but I still think it's useful to be explicit that "adding a new language" involves some easy/automated steps but also a lot of skilled human work too